lib/BaseStripeClient.php

```diff
@@ -18,6 +18,7 @@ class BaseStripeClient implements StripeClientInterface, StripeStreamingClientIn
         'api_key' => null,
         'client_id' => null,
         'stripe_account' => null,
+        // Note, even if null, ApiRequestor will default this to Stripe::$apiVersion
```

Comment:
isn't that wrong? Should StripeClient inherit global settings?

Reply:
We can change the behavior so that StripeClient initialized without an explicit stripe_version sends requests using the pinned version instead of `Stripe::$apiVersion` but that change is breaking.

Reply:
e.g.
```php
Stripe::setApiVersion("2020-XX-YYYY");
$stripe = new StripeClient("sk_...");
$stripe->customers->create(...);
```
will use a different API version if we change the fallback behavior.

Are you suggesting we make that change now?

Reply:
I opened https://github.com/stripe/stripe-php/pull/1551 in response to our convo here. I'm not sure we should merge it along with the major.